__tests__/jsonPlaceholder.test.ts

```diff
@@ -1,7 +1,10 @@
-import { KoaContext } from "../types";
+import { AppRouterContext } from "../interface";
 import { JsonPlaceHolder } from "../typescriptRoutes/jsonPlaceholder";
 import { MockHttpClient } from "../utils/MockHttp";
 
+afterEach(() => {
+    jest.resetAllMocks();
+});
 describe("Json PlaceHolder test ", () => {
     test("should be able to check the jsonPlaceholder handleGetRequest function ", async () => {
```

Comment:
read about api naming convention and try to adopt those naming conventionn afterwards.
I think react function naming convention and rest api naming conventions are different.